tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -528,6 +531,18 @@ class ExecutionTest {
                        L"Table:ShaderOpArithTable.xml#PackUnpackOpTable")
   END_TEST_METHOD()
 
+  BEGIN_TEST_METHOD(WaveMatrixLoadStoreTests)
+  TEST_METHOD_PROPERTY(L"Priority", L"2")
+  END_TEST_METHOD()
+
+  BEGIN_TEST_METHOD(WaveMatrixScalarTests)
+  TEST_METHOD_PROPERTY(L"Priority", L"2")
+  END_TEST_METHOD()
+
+  BEGIN_TEST_METHOD(WaveMatrixMathTests)
+  TEST_METHOD_PROPERTY(L"Priority", L"2")
```

Comment:
Should these be priority 2? IIRC, that means they don't run by default

Reply:
The original pattern was to use priority to disable execution tests for next-shader-model features to prevent them from running as part of regular runs, because they require a specific prerelease runtime, device, and prerelease driver to execute successfully.  They are run with specific command lines to target this functionality when in the right operating environment.  We could discuss whether this is still necessary for these tests, given that we now have the ability to query feature support from the API.

Reply:
Priority 2 is no longer needed, the tests check if WaveMMA is supported and will be Skipped if not.